src/test/crypto_tests.cpp

```diff
@@ -585,6 +586,135 @@ BOOST_AUTO_TEST_CASE(hkdf_hmac_sha256_l32_tests)
                 "8da4e775a563c18f715f802a063c5a31b8a11f5c5ee1879ec3454e5f3c738d2d");
 }
 
+#define XOR(v, w) ((v) ^ (w))
```

Comment:
What is the advantage of defining `XOR` as a macro here ?

Reply:
I thought it improves readability (got inspired by DJBs code). But happy to remove it since it saves space.

Reply:
It doesn't make the code more readable to me, at least. I recognize the C operator but the macro I had to look up. But I don't feel strongly about it.

Reply:
Removed the extra XOR macro.

Reply:
🤔  there are still 6 `XOR` usages, is this intended?